src/distdl/backends/mpi/autograd/transpose.py

```diff
@@ -148,7 +153,7 @@ def forward(ctx, input, P_union, x_global_structure,
             input_numpy = input.detach().numpy()
             for (sl, sz, partner), buff in zip(P_x_to_y_overlaps, P_x_to_y_buffers):
                 if buff is not None:
-                    np.copyto(buff, input_numpy[tuple(sl)].ravel())
+                    np.copyto(buff, input_numpy[sl].ravel())
```

Comment:
I assume sl is guaranteed to be a tuple here? List indexing is being deprecated but it should be fine if so.

Reply:
yeah, I updated assemble_slices to return a tuple, to account for this.  it should have done that anyway